base/replutil.jl

```diff
@@ -444,6 +456,10 @@ function show_method_candidates(io::IO, ex::MethodError, kwargs::Vector=Any[])
             buf = IOBuffer()
             tv = Any[]
             sig0 = method.sig
+            # @Hardcoded: don't display the sysimg.jl method
```

Comment:
Reduces programmer friction; the `@` and the capital 'H' make it much more likely you _only_ find what you're actually searching for. (i.e: the tag.)

You could also do things like `#Hardcode: ...`, since this is Julia, but that can blend in with ordinary comments a bit too easily for my liking; the `@` makes it clearer that it is _actually_ a tag.

You can also do things like, `# @Cleanup @Cleanup @Cleanup` etc, if you want it to _really_ stand out when you search for it. >)